src/modules/Telemetry/Sensor/RAK9154Sensor.cpp

```diff
@@ -0,0 +1,191 @@
+#ifdef HAS_RAKPROT 
```

Comment:
I don't think the RAK9154 support is places correctly. The Telemetry folder is for devices that actually transmit values over Lora. This looks more like gathering values for internal processing. If this is only ever used once for this particular device, you can as well put it in the variant folder and skip deriving it from TelemetryModule. If you add the OSThread class to inheritance you can use the runOnce model without having to define it as a sensor (which it isn't in the meshtastic sense)